server/src/IncidentSender.cc

```diff
@@ -111,6 +111,7 @@ struct IncidentSender::Impl
 	AtomicValue<bool> trackerChanged;
 	std::mutex   trackerLock;
 	HatoholError lastResult;
+	bool shouldRecordIncidentHistory;
```

Comment:
I've added it: https://github.com/project-hatohol/hatohol/pull/2266/files#diff-ab1d8e32a3713e346c9c43936482ca31R120